client/scripts/index.js

```diff
@@ -19,9 +23,23 @@ function getPlayerColorFromNumber(player_number) {
     return '#' + colors[player_number]
 
 }
+// allows player interations with interface
+function enablePlayerActions(data){
```

Comment:
why is data in the args

Reply:
i think it important to denote that the function is able to receive data from the server.
Im still thinking theres an implementation where the server passes the pad object back to the client
with that being how we control scope